api/app/settings/common.py

```diff
@@ -310,6 +310,7 @@
     "X-Environment-Key",
     "X-E2E-Test-Auth-Token",
     "sentry-trace",
+    "baggage",
```

Comment:
`baggage` seems like a very generic header name 🤔 . I'm thinking we could improve the logic here in a couple of ways: 

1. Only add these exceptions if Sentry is enabled. Something like the following: 

```python
if SENTRY_SDK_DSN:
    CORS_ALLOW_HEADERS = CORS_ALLOW_HEADERS + ("sentry-trace", "baggage")
```

2. Allow the user to configure the headers themselves by adding a new environment variable: 

```python
FLAGSMITH_CORS_EXTRA_ALLOW_HEADERS = env.list("FLAGSMITH_CORS_EXTRA_ALLOW_HEADERS", default=[])

CORS_ALLOW_HEADERS = [
    *default_headers, 
    *FLAGSMITH_CORS_EXTRA_ALLOW_HEADERS,
    "X-Environment-Key", 
    "X-E2E-Test-Auth-Token",
]
```

Or maybe both of the above would be sensible?

Reply:
Without looking at the code isn't SENTRY_SDK_DSN a flag for flagsmith itself.

The issue I'm having is in my frontend library I'm using sentry-JavaScript and they keep hacking more headers onto all outgoing requests. Everytime they do this it breaks flagsmith. 

I like the second method you propose. I can add that in here

Reply:
>Without looking at the code isn't SENTRY_SDK_DSN a flag for flagsmith itself.
>
> The issue I'm having is in my frontend library I'm using sentry-JavaScript and they keep hacking more headers onto all outgoing requests. Everytime they do this it breaks flagsmith.

Ah, yes, of course! That makes sense. 

>I like the second method you propose. I can add that in here

Great, thanks @tm1000 👌